src/rules/selector-class-pattern/index.js

```diff
@@ -39,6 +39,12 @@ export default function (pattern, options) {
       // Ignore Sass intepolation possibilities
       if (/#{.+}/.test(rule.selector)) { return }
 
+      // Ignored called Less mixins
+      if (rule.ruleWithoutBody) { return }
+
+      // Ignored non-ouputting Less mixin definitions
+      if (_.endsWith(rule.selector, ")")) { return }
```

Comment:
Thanks for the commenting --- this would be tricky to remember the reasoning for otherwise.